radio/src/pulses/pulses_avr.cpp

```diff
@@ -110,58 +110,55 @@ uint8_t *pulses2MHzWPtr = pulses2MHz;
 #define CTRL_REP_1CMD -3
 #define CTRL_REP_2CMD -6
 
-#define SETUP_PULSES_DURATION 1000/*500us*/
+#define SETUP_PULSES_DURATION 1000 // 500us
+uint8_t g_ppmPulsePolarity = 0; // Needed for Bit-bang PPM.
 
-// TIMER1_COMPA_vect used for PPM and DSM2=SERIAL
-uint8_t g_ppmPulsePolarity = 0;
-ISR(TIMER1_COMPA_vect) //2MHz pulse generation (BLOCKING ISR)
+// TIMER1_COMPA_vect used for PPM and DSM2=SERIAL.
+ISR(TIMER1_COMPA_vect) // 2MHz pulse generation (BLOCKING ISR).
 {
-  uint8_t dt = TCNT1L; // record Timer1 latency for DEBUG stats display
-  
-  // Call setupPulses only after REST pulse had been sent.
-  // Must do this before toggle PORTB to keep timing accurate
+  uint8_t dt = TCNT1L; // Record Timer1 latency for DEBUG stats display.
+
+  // Call setupPulses only after "rest" period has elapsed.
+  // Must do this before toggle PORTB to keep timing accurate.
   if (IS_DSM2_SERIAL_PROTOCOL(s_current_protocol[0]) || *((uint16_t*)pulses2MHzRPtr) == 0) {
     if (!IS_DSM2_SERIAL_PROTOCOL(s_current_protocol[0])) {
       OCR1A = SETUP_PULSES_DURATION;
+#if defined(CPUM2560) // CPUM2560 hardware toggled PPM out.
+      OCR1B = OCR1A;
+      if ( g_model.pulsePol ) TCCR1A = (TCCR1A | (1<<COM1B1)) & ~(1<<COM1B0); // Set idle level.
```

Comment:
Cosmetics, I would prefer that the test is written differently:
if (xxx) // no spaces
  TCCR1A ...
else
  TCCR1A ...